### ISSUE TITLE
CORS from github?

### ISSUE THREAD

**Issue by CKGrafico** [adorableio/avatars-api-middleware#81]
I'm trying to use this api from another github projects but I've CORS error:

Failed to load https://api.adorable.io/avatars/list: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://ckgrafico.github.io' is therefore not allowed access.

**Comment by bigtiger** [comment]
@CKGrafico Are you still seeing this error?

**Comment by CKGrafico** [comment]
Yes :( https://ckgrafico.github.io/FooCards-Nuxt/login (click the button)
